service/src/main/java/bio/terra/profile/app/configuration/BeanConfig.java

```diff
@@ -14,6 +15,13 @@
 
 @Configuration
 public class BeanConfig {
+  private static final String CLIENT_NAME = "profile";
+
+  @Bean("crlClientConfig")
+  public ClientConfig crlClientConfig() {
```

Comment:
This was the sole shared resource between the Azure parts of `CrlService` and the GCP part. 
Other constructor parameters of the services are set up as beans, so it made sense to do the same here.

Reply:
I moved this piece from `CrlService`; the only change I made was to make it a bean.

The difference is that if we needed to use Janitor, we'd need to add the `CleanupConfig`.

Reply:
Could you update the comment to clarify that the thing it doesn't need is `CleanupConfig`?